examples/custom-node/Cargo.toml

```diff
@@ -19,11 +19,126 @@ reth-rpc-api.workspace = true
 reth-rpc-engine-api.workspace = true
 reth-ethereum = { workspace = true, features = ["node-api", "network", "evm", "pool"] }
 
+reth-basic-payload-builder.workspace = true
```

Comment:
since we intend this to be a blueprint, we should cleanup deps here, let's try to pull in as much as possible from `reth-ethereum` L20 and add missing re-exports

so ideally none of these imports are necessary